projects/serializers.py

```diff
@@ -179,6 +178,5 @@ class Meta:
             "status",
             "control",
             "project",
-            "remarks",
```

Comment:
This as well; was it intentional to remove this?

Reply:
Got it